Formula/pastebinit.rb

```diff
@@ -19,6 +11,7 @@ def install
     inreplace "pastebinit", "/usr/bin/python3", Formula["python3"].opt_bin + "python3"
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
+    rm_r etc/"pastebin.d"
```

Comment:
I don't have a `pastebin` key, and I get `Bad API request, invalid api_dev_key` even on version `1.5`.
```sh
$ pastebinit -v
pastebinit v1.5
```

I guess, you need to set your personal `api_dev_key` in the configuration file. If this installation process, removes
`rm_r etc/"pastebin.d"`, users will have to reconfigure their api keys again.